core/classes/HookHandler.php

```diff
@@ -64,11 +64,17 @@ public static function executeEvent($event, $param = null){
 		// Execute user made webhooks
 		foreach(self::$_hooks as $hook) {
 			if(in_array($event, $hook['events'])) {
-				$param['webhook'] = $hook['url'];
-				call_user_func($hook['action'], $param);
-				
-			}
-		}
+                if ($event == 'newTopic') {
```

Comment:
The indentation looks a bit off here, because of mismatched tabs/spaces. I am not sure which one is preferred for this project, @samerton ?

Reply:
Code is fine in IDE, Github must be displaying it incorrectly 🤔

Reply:
Your IDE probably displays tabs as 4 spaces which makes it look fine in combination with actual 4 space indentation. GitHub by default displays tabs as 8 spaces